dns/route53/dns.go

```diff
@@ -36,23 +36,25 @@ func NewSRVManager(client *Client, hostedZoneID string, recordName string, ttl u
 
 // edit provides both add and removal capabilities
 func (s *SRVManager) edit(add bool, srv *net.SRV) error {
-	log.Infof("Looking up SRV resource record set for %s", s.recordName)
-	currentResourceRecordSet, err := s.client.GetResourceRecordSetByName(s.hostedZoneID, s.recordName, "SRV")
+	recordName := addDotSuffixIfNeeded(s.recordName)
```

Comment:
This was present in `GetResourceRecordSetByName` already, now we do it twice - did you intend to move it from there to here, and forgot to remove it there?

Reply:
It still in present in `GetResourceRecordSetByName`, but the suffix check is necessary here, too.

All route53 client calls are abstracted in `route53/client.go`, except `SRVManager.client.Service.ChangeResourceRecordSets` in `SRVManager.edit(...)` which is located in [`route53/dns.go`](https://github.com/wireapp/srv-announcer/blob/master/dns/route53/dns.go#75), though. So, when calling `SRVManager.Add(...)` or `SRVManager.Remove(...)` this normalization step does not happen. I considered to introduce another method to `Client` in order to abstract the call of `ChangeResourceRecordSets` and use the abstracted on in `SRVManager.edit(...)` instead, but I figured, this kind of refactoring is out of scope for this PR. So I added the normalization call here instead, like it's done in all `Client` methods.

Would you suggest another approach?

Reply:
But we *do* immediately call `GetResourceRecordSetByName`, and all code below did use `s.recordName`, which came back in whatever format it was from the r53 API.

Reply:
has this been updated?